src/tpm.rs

```diff
@@ -1,11 +1,30 @@
+use std::collections::HashMap;
 use std::str::FromStr;
-
 use tss_esapi::constants::algorithm::HashingAlgorithm;
-use tss_esapi::constants::tss as tss_constants;
+use tss_esapi::constants::tss::*;
 use tss_esapi::tss2_esys::{ESYS_TR, ESYS_TR_NONE, ESYS_TR_RH_OWNER};
 use tss_esapi::Context;
 use tss_esapi::Tcti;
 
+fn int_to_str(num: u32) -> String {
```

Comment:
Well, it's not really a standard integer-to-string operation. It's basically parsing every byte in a `u32` (i.e. up to 4 bytes) as a different hex character. Because that's how TPM does strings.

Reply:
(Renaming to `int_to_string` is reasonable though)